File: specification/resources/resource-manager/Microsoft.Solutions/stable/2018-06-01/examples-js/getApplicationById.js
const { ApplicationClient } = require("@azure/arm-managedapplications");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Gets the managed application.
 *
 * @summary Gets the managed application.
 * x-ms-original-file: specification/resources/resource-manager/Microsoft.Solutions/stable/2018-06-01/examples/getApplicationById.json
 */
async function getApplicationById() {
  const subscriptionId = "00000000-0000-0000-0000-000000000000";
  const applicationId = "myApplicationId";
  const credential = new DefaultAzureCredential();
  const client = new ApplicationClient(credential, subscriptionId);
  const result = await client.applications.getById(applicationId);
  console.log(result);
}

getApplicationById().catch(console.error);

File: specification/resources/resource-manager/Microsoft.Solutions/stable/2018-06-01/examples-java/getApplicationById.java
/** Samples for Applications GetById. */
public final class Main {
    /*
     * x-ms-original-file: specification/resources/resource-manager/Microsoft.Solutions/stable/2018-06-01/examples/getApplicationById.json
     */
    /**
     * Sample code: Get application by id.
     *
     * @param manager Entry point to ApplicationManager.
     */
    public static void getApplicationById(com.azure.resourcemanager.managedapplications.ApplicationManager manager) {
        manager.applications().getByIdWithResponse("myApplicationId", com.azure.core.util.Context.NONE);
    }
}
